docs/guide/blog/comment-blog.md

```diff
@@ -0,0 +1,619 @@
+
+This tutorial relies on the `blog` blockchain that you built in the build a blog tutorial. 
+
+## Fetch Functions Using List Command
+
+To get the useful functions for this tutorial, you use the `starport scaffold list comment --no-message` command. Make sure to familiarize yourself with the command.
+
+1. Navigate to the `blog` directory that you created in the build a blog tutorial.
+
+2. To create the source code files to add CRUD (create, read, update, and delete) functionality for data stored as an array, run:
+
+```bash
+starport scaffold list comment --no-message
+```
+
+The `--no-message` flag disables CRUD interaction messages scaffolding because you will write your own messages.
+
+The command output shows the files that were created and modified:
+
+```text
+create proto/blog/comment.proto
+modify proto/blog/genesis.proto
+modify proto/blog/query.proto
+modify vue/src/views/Types.vue
+modify x/blog/client/cli/query.go
+create x/blog/client/cli/query_comment.go
+create x/blog/client/cli/query_comment_test.go
+modify x/blog/genesis.go
+modify x/blog/genesis_test.go
+create x/blog/keeper/comment.go
+create x/blog/keeper/comment_test.go
+create x/blog/keeper/grpc_query_comment.go
+create x/blog/keeper/grpc_query_comment_test.go
+modify x/blog/module.go
+modify x/blog/types/genesis.go
+modify x/blog/types/genesis_test.go
+modify x/blog/types/keys.go
+
+🎉 comment added.
+```
+
+
+## Add a Comment to a Post
+
+To create a new message that adds a comment to the existing post, run:
+
+```bash
+starport scaffold message create-comment postID:uint title body
+```
+
+The `starport scaffold message` command accepts `postID` and a list of fields as arguments. The fields are `title` and `body`.
+
+Here, `postID` is the reference to previously created blog post.
+
+The `message` command has created and modified several files:
+
+```text
+modify proto/blog/tx.proto
+modify x/blog/client/cli/tx.go
+create x/blog/client/cli/tx_create_comment.go
+modify x/blog/handler.go
+create x/blog/keeper/msg_server_create_comment.go
+modify x/blog/types/codec.go
+create x/blog/types/message_create_comment.go
+create x/blog/types/message_create_comment_test.go
+
+🎉 Created a message `create-comment`.
+```
+
+As always, start your development with a proto file. 
+
+In the `proto/blog/tx.proto` file, edit `MsgCreateComment` to:
+
+- Add `createdAt`
+- Define the `id` for `message MsgCreateCommentResponse`:
+
+```go
+message MsgCreateComment {
+  string creator = 1;
+  int32 blogID = 2;
+  string title = 3;
+  string body = 4;
+  uint64 id = 5;
+}
+
+message MsgCreateCommentResponse {
+  uint64 id = 1;
+}
+```
+
+ You see in the `proto/blog/tx.proto` file that the `MsgCreateComment` has five fields: creator, title, body, blogID, and id. Since the purpose of the `MsgCreateComment` message is to create new comments in the store, the only thing the message needs to return is an ID of a created comments. The `CreateComment` rpc was already added to the `Msg` service:
```

Comment:
Where is `blogID` coming from?

Reply:
Yeah..  seems like it should be `postID`. Searched the PR further I see `postID` being used everywhere.

Reply:
@alijnmerchant21 did we want `postID` here? 
```suggestion
 You see in the `proto/blog/tx.proto` file that the `MsgCreateComment` has five fields: creator, title, body, blogID, and id. Since the purpose of the `MsgCreateComment` message is to create new comments in the store, the only thing the message needs to return is an ID of a created comments. The `CreateComment` rpc was already added to the `Msg` service:
```